test/e2e/k8s_util.go

```diff
@@ -87,46 +87,57 @@ func (k *KubernetesUtils) GetPodsByLabel(ns string, key string, val string) ([]v
 func (k *KubernetesUtils) probe(
 	pod *v1.Pod,
 	podName string,
+	containerName string,
 	dstAddr string,
 	dstName string,
 	port int32,
 	protocol v1.Protocol,
 ) PodConnectivityMark {
+	protocolStr := map[v1.Protocol]string{
+		v1.ProtocolTCP:  "tcp",
+		v1.ProtocolUDP:  "udp",
+		v1.ProtocolSCTP: "sctp",
+	}
```

Comment:
Just found you did this exactly but changed because of https://github.com/antrea-io/antrea/pull/3006#discussion_r745907281. I don't have strong opinion, so feel free to ignore this comment.